website/content/en/docs/concepts/provisioners.md

```diff
@@ -58,7 +58,7 @@ spec:
     example.com/owner: "my-team"
 
   # Requirements that constrain the parameters of provisioned nodes.
-  # These requirements are combined with pod.spec.affinity.nodeAffinity rules.
+  # These requirements are combined with pod.spec.affinity.nodeAffinity and pod.spec.nodeSelector rules.
```

Comment:
```suggestion
  # These requirements are combined with pod.spec.topologySpreadConstraints, pod.spec.affinity.nodeAffinity, pod.spec.affinity.podAffinity, and pod.spec.nodeSelector rules.
```

I figure if we're touching this, we might as well add all the places that requirements come from

Reply:
ya makes sense

Reply:
ya makes sense